cloud/services/security/securitygroup.go

```diff
@@ -157,6 +157,9 @@ func (s *Service) CreateSecurityGroupRule(securityGroupId string, flow string, i
 		securityGroupRuleResponse, httpRes, err = oscApiClient.SecurityGroupRuleApi.CreateSecurityGroupRule(oscAuthClient).CreateSecurityGroupRuleRequest(createSecurityGroupRuleRequest).Execute()
 		if err != nil {
 			if httpRes != nil {
+				if httpRes.StatusCode == 409 {
+					return true, err
```

Comment:
```suggestion
					return true, nil
```
The error should not be returned, as in places where we use this function we check with `if err != nil`

Reply:
We could add a log though if we want to catch it (knowing this is a quickfix for an Outscale api bug)